README.md

```diff
@@ -41,7 +41,7 @@ const LIBCLANG_HEADERS = [joinpath(LIBCLANG_INCLUDE, header) for header in readd
 wc = init(; headers = LIBCLANG_HEADERS,
             output_file = joinpath(@__DIR__, "libclang_api.jl"),
             common_file = joinpath(@__DIR__, "libclang_common.jl"),
-            clang_includes = vcat(LIBCLANG_INCLUDE, CLANG_INCLUDE),
+            clang_includes = [LIBCLANG_INCLUDE],
```

Comment:
Could you elaborate more about why `CLANG_INCLUDE` needs to be deleted?

Reply:
My bad, I thought `CLANG_INCLUDE` was a typo.